contracts/SafEth/derivatives/Reth.sol

```diff
@@ -41,8 +41,8 @@ contract Reth is IDerivative, Initializable, OwnableUpgradeable {
     */
     function initialize(address _owner) external initializer {
         _transferOwnership(_owner);
-        maxSlippage = (1 * 10 ** 16); // 1%
-    }
+   uint256 constant private MAX_SLIPPAGE = 10000; // 1% represented as parts per million (PPM)
```

Comment:
define a constant MAX_SLIPPAGE that represents the maximum allowed slippage as 1% expressed as parts per million (PPM). We then set the maxSlippage variable to the value of MAX_SLIPPAGE. By using a constant value and expressing the slippage as PPM, we can avoid potential errors due to decimal places and make the code easier to read and understand.